src/components/ListCard/ListCard.jsx

```diff
@@ -41,7 +42,25 @@ const StyledContentWrapper = styled.div`
   }
 `;
 
-const ListCard = ({ id, title, size, data, isLoading, loadData, hasMoreData, ...others }) => {
+const StyledValueSpan = styled.div`
```

Comment:
I was recently told that we should move away from the `styled` approach and use scss.

Reply:
Yes; although since this component was written with styled-components and I didn't want to refactor it to use scss (at this time), or use a mixture of both in the same component, I continued the `styled` approach for my changes.